### ISSUE TITLE
Help request on setting the visual editor in a modal box

### ISSUE THREAD

**Issue by Teqa** [andrewtelnov/survey.js.editor#2]
A common scenario (issue) in bootstrap is the usage of multiple modals. 
When a modal is triggered from another modal, this will get a mixed backdrop. Very confusing for all bootstrap users. Solution can be found on several forums:

`// Fix stacked models in bootstrap 
$(document)  
    .on('show.bs.modal', '.modal', function(event) {
        $(this).appendTo($('body'));
    })
    .on('shown.bs.modal', '.modal.in', function(event) {
        setModalsAndBackdropsOrder();
    })
    .on('hidden.bs.modal', '.modal', function(event) {
        setModalsAndBackdropsOrder();
    })
    .on('show.bs.tooltip', '', function(event) {
        $('.tooltip').not(this).hide();
    });
    

function setModalsAndBackdropsOrder() {  
    var modalZIndex = 1040;
    $('.modal.in').each(function(index) {
        var $modal = $(this);
        modalZIndex++;
        $modal.css('zIndex', modalZIndex);
        $modal.next('.modal-backdrop.in').addClass('hidden').css('zIndex', modalZIndex - 1);
    });
    $('.modal.in:visible:last').focus().next('.modal-backdrop.in').removeClass('hidden');
}`

However, this solution does not seem to work with the architecture of the visual editor. 
Within the visual editor, there are several modules, like the one for editing options. 
It seems this solution detects these modals as already opened. This is causing the modal to disappear. Maybe it is opened from the start and just made visible?
Would it be possible to change the visual editor so that it actually (re) opens the modals?

**Comment by andrewtelnov** [comment]
Yes, the first item in the search gives me this: http://www.bootply.com/61322
I will take a look at it. By the way, is it possible to show Visual Editor on the page itself, like having tabs?

To be honest, I do not want to spend to much time on this issue right now. There are many other issues that I have to address :(

Thank you,
Andrew Telnov

**Comment by Teqa** [comment]
Agree with priority. It is nice to have this in editor in a modal, but works just fine in a full page. 
Leave it open with low prio....
